### PR TITLE
CNV-4013 Update documentation layout of the vm screens 

### PR DESCRIPTION
**Pull request by bgaydosrh**
Test Build: http://file.bos.redhat.com/bgaydos/063020/virt/

Tagging @jelkosz for code review
Tagging @nellyc for QE resource (will send email as well).

Please label *peer review needed* and *enterprise-4.5* 

Thanks
Bob

### CODE REVIEW

File: modules/virt-cancelling-vm-migration-web.adoc [hunk 1]
```diff
@@ -5,15 +5,19 @@
...
-. You can cancel the migration from this screen, which makes it easier to perform actions on multiple virtual machines in the one screen, or from the *Virtual Machine Details* screen where you can view comprehensive details of the selected virtual machine:
+. In the {VirtProductName} console, click *Virtualization* from the side menu.
+. Click the *Virtual Machine* tab to display your virtual machines.
+. Cancel migration of your virtual machines. Using the *Virtual Machine* tab,
+you can perform actions on multiple virtual machines, or from the
+*Virtual Machine Details* screen where you can view comprehensive details
+of the selected virtual machine:
 ** Click the Options menu {kebab} at the end of virtual machine and select
 *Cancel Virtual Machine Migration*.
 ** Click the virtual machine name to open the *Virtual Machine Details*
```
Comment (gouyang) [inline modules/virt-cancelling-vm-migration-web.adoc:23]: The first tab of the virtual machine is `Overview` on OCP 4.5, `Details` is the 2nd tab. Click the virtual machine name is opening the `Virtual Machine Overview`. The action menu is available on all tabs.

File: modules/virt-editing-vm-web.adoc [hunk 2]
```diff
@@ -28,18 +28,19 @@ endif::[]
...
 .Procedure
 
-. Click *Workloads* -> *{object-gui}s* from the side menu.
-. Select a {object} to open the *{object-gui} Overview* screen.
-. Click the pencil icon to make that field editable.
+. Click *Virtualization* from the side menu.
+. Select the {object} tab to open the *{object-gui} Overview* screen.
+. Select the *{object}* you want to edit.
+. Click the pencil icon to make a field editable.
 . Make the relevant changes and click *Save*.
```
Comment (gouyang) [inline modules/virt-editing-vm-web.adoc:40]: VM overview tab serves as the VM dashboard, no fields are editable on overview. `Details` tab has editable fields, it requires the VM is stopped on OCP 4.5.
Suggest steps:
1. Click *Virtualization* from the side menu.
2. Select the *Virtual Machine* tab.
3. Click the virtual machine name to open the *Virtual Machine Overview*.
4. Click “Details" tab.
5. Select the *{object}* you want to edit.
6. Click the pencil icon to make a field editable.
7. Make the relevant changes and click *Save*.

**Comment by jelkosz** [comment]
Agree with what @gouyang wrote, other than that, looks good to me

**Comment by bgaydosrh** [comment]
PR replaced by https://github.com/openshift/openshift-docs/pull/23885

All changes suggested in this PR have been implemented in https://github.com/openshift/openshift-docs/pull/23885

Closing and deleting unmerged branch.

cc: @sjhala-ccs @gouyang @jelkosz

**Pull request closed without merging**